packages/dx-react-scheduler/docs/guides/grouping.md

```diff
@@ -48,7 +45,7 @@ Users can drag-and-drop appointments between groups or use the appointment edit
 
 Timetable cells have the `groupingInfo` property that lists groups to which a particular cell belongs. Use this property to customize cells depending on their group.
```

Comment:
Line 26:
Change "one _under_ another" to "one _underneath_ another"
For the third sentence, can we say:
"You can adjust group orientation based on a view, as shown in the following example:"

Line 34:
Change to: In the example below, appointments are first grouped by date and then by resource. Click the "Group by Date First" switch to change the order.

Line 46:
Change "the `groupingInfo` property" to "a `groupingInfo` property"
Change "depending on their group" to "based on their group"